planner/core/exhaust_physical_plans.go

```diff
@@ -704,6 +704,8 @@ func (p *LogicalJoin) constructInnerIndexScanTask(
 		KeepOrder:        keepOrder,
 		Ranges:           ranger.FullRange(),
 		rangeInfo:        rangeInfo,
+		isPartition:      ds.isPartition,
```

Comment:
Seems that it is also not set in `(b *PlanBuilder) buildPhysicalIndexLookUpReader`, should we fix it in this pr?

Reply:
It looks like `admin check table` and `admin check index` does not work for partitioned table at all? I didn't find code converting partitioned table to `union all` for these 2 command, @crazycs520 PTAL

Reply:
Yes. `admin check table` and `admin check index` call `buildPhysicalIndexLookUpReader` directly. I think they don't set `isPartition`